src/main/resources/init_db.sql

```diff
@@ -18,3 +18,22 @@ CREATE TABLE `drivers` (
                                   PRIMARY KEY (`id`),
                                   UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
                                   UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+
```

Comment:
your db name taxi_service_db or taxi_service ?
delete empty space , and file will be seems better